tox.ini

```diff
@@ -33,8 +33,9 @@ passenv =
 # {posargs} can be passed in by additional arguments specified when invoking tox.
 # Can be used to specify which tests to run, e.g.: tox -- -s
 commands =
-    coverage run --source sagemaker_containers -m py.test {posargs}
+    coverage run --source sagemaker_container_support -m py.test {posargs}
```

Comment:
we need to fix project name, package name, and source code layout so they are aligned. since the repo is called `sagemaker-containers`, the package name should be `sagemaker_containers` and the directory structure should be `src/sagemaker_containers`

but **important** do that in separate PR to avoid the features / file moves combo.

Reply:
I have another branch with the name changes waiting for this one